src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,8 @@
 package mate.academy.lib;
 
-public @interface Inject {
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
 
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Inject {
```

Comment:
It would be great to add a Target annotation here to indicate that we will put our own annotation only on fields